### ISSUE TITLE
No `owl:inverseOf` or other inter-Resource relations defined in Ontolex schemas

### ISSUE THREAD

**Issue by simongray** [globalwordnet/schemas#53]
Most of the relations described at https://globalwordnet.github.io/gwadoc/ have inverse relations (called "reverse" in the documentation). However, none of these have been marked as `owl:inverseOf` in the Ontolex schema files. I wonder why that is the case? I really hope it's simply a case of a lack of time or an oversight, rather than a conscious omission.

There _is_ a [wn-simple-1.1.ttl](https://github.com/globalwordnet/schemas/blob/master/wn-simple-1.1.ttl) which seems to be a pre-Ontolex RDF schema and this _does_ retain some useful OWL definitions for inter-Resource relations, including `owl:inverseOf`. However, it seems that this is fundamentally incompatible with the later Ontolex versions (e.g. `wn-lemon-1.1.ttl` or `wn-lemon-1.1.rdf`) -- or at least I see no path to obtain equivalence between the different ontology definitions. If I choose the Ontolex schemas I gain a broader set of relations and build on the most current shared definition of what constitutes a WordNet graph, but on the other hand I lose very valuable OWL information. In my case, I really need both.

I am currently using the 1.1 schema files to construct an ontology for DanNet. I actually expected the various relations to have e.g. `owl:inverseOf` since this is so heavily featured in the documentation (almost every relation listed has a "reverse" relation) so I never bothered to check the schemas if they actually contained this information. I am relying on OWL reasoning to infer the "missing" RDF triples in order to have a consistent dataset and less duplication of data. The fact that the Ontolex-schemas have relatively sparse ontological information compared to `wn-simple-1.1.ttl` is really not a great foundation for us at the Center for Language Technology to build a knowledge graph on. The schemas are too simple in their current incarnations.

I appreciate the work you're doing to consolidate WordNet standards and create a common foundation, so I hope that you will reconsider.

**Comment by simongray** [comment]
I could  of course construct my own elaborate DanNet Ontology and define DanNet-specific relations that contain all of the relevant OWL data, but I think it's such a shame that this sort of information is not readily at hand in the GWA Ontolex schemas, which means it isn't known or shared between datasets using these schemas. I would much prefer that something like `owl:inverseOf` be made part of the official GWA schemas, especially considering that you already imply that these relationships exist through your documentation, they're just not codified.

**Comment by arademaker** [comment]
Hi @simongray , I agree that it is important to better explain the models and the reason for such different approaches. The so-called `wn-simple.ttl` is based on https://www.w3.org/TR/wordnet-rdf/ and I have been using it for the Portuguese Wordnet (http://openwordnet-pt.org) since its first release, years before the Ontolex schema. As you pointed out, I did pay attention to maximise the use of axioms and OWL constraints mainly during the validation work we did in the paper "Verifying Integrity Constraints of an RDF-based WordNet" (https://aclanthology.org/2016.gwc-1.44/). 

But currently, the page https://globalwordnet.github.io/schemas/#rdf does not clearly explain the two vocabularies and their relation. Moreover, this page is not completely in sync with https://globalwordnet.github.io/gwadoc/, which focuses on the relations nor in their encoding in concrete formats (XML, RDF, etc). 

To sum up, we may need help to improve the current state of the repositories. Would you be willing to help? Maybe we can start to discuss concrete possible actions.

**Comment by simongray** [comment]
To start with, I guess I can add in the `owl:inverseOf` relations, but I'm not sure about the formalities involved and how you would handle external contributions. There is a version 1.0 and a version 1.1, for example. Would this go into 1.1 or a version 1.1.1 or 1.2? Things like that.

Also, I'm a software developer, not a linguist, so my domain knowledge is probably limited compared to your group. I mostly just notice inconsistencies like this since I'm using the schemas.

**Comment by jmccrae** [comment]
I agree with adding the inverse relations. This should be an easy enough fix.

Version 1.1 is complete so I will create a new branch for the 1.2 update

**Comment by simongray** [comment]
(the PR in question is #54)

**Comment by simongray** [comment]
Actual PR for the changes made to fix this issue: https://github.com/globalwordnet/schemas/pull/57

**Comment by simongray** [comment]
Thank you for merging this, @jmccrae.

I will likely have a few more suggested changes for the 1.2 release. I'll make the appropriate issues + PRs for them of course.

**Comment by jmccrae** [comment]
Thanks for your contribution, we welcome more changes!
